### PR TITLE
Update dependencies and readme

### PR DESCRIPTION
**Pull request by jayudey-wf**
## Issue
- Some of our dependencies needed to be updated or could be removed
- The `saucelabs` task readme section needed to be updated
## Changes

**Source:**
- Updated dependencies, readme

**Tests:**
- n/a
## Areas of Regression
- failing tests
- `saucelabs task`
## Testing
- passing CI
- verify `saucelabs` task is still working as expected
## Code Review

@trentgrover-wf
@maxwellpeterson-wf
@evanweible-wf
@dustinlessard-wf

### CODE REVIEW

File: lib/src/tasks/saucelabs/sauce_iframe_runner/sauce_iframe_runner.dart [hunk 1]
```diff
@@ -300,8 +300,8 @@ Future<List<String>> runTests(String testUrl, String platform,
 
   // Load the tests
 
-  var augmentedTestUrl = new Fluri(testUrl)
-    ..setQueryParam('platform', platform);
+  var augmentedTestUrl = new UriBuilder.fromUri(Uri.parse(testUrl))
+    ..queryParameters = {'platform': platform};
```
Comment (evanweible-wf) [inline lib/src/tasks/saucelabs/sauce_iframe_runner/sauce_iframe_runner.dart]: There's no need for a uri package at all here, the builtin `Uri` class will suffice:

```dart
var augmentedTestUrl = Uri.parse(testUrl).replace(queryParameters: {'platform': platform});
```

File: pubspec.yaml [hunk 2]
```diff
@@ -15,16 +15,16 @@ dependencies:
...
   dart_style: ">=0.2.4 <0.3.0"
   dartdoc: "^0.9.0"
-  fluri: "^1.1.1"
   path: "^1.3.6"
   rate_limit: "^0.1.0"
   resource: ">=1.0.0 <3.0.0"
   sass: "^0.4.2"
   test: "^0.12.0"
   uuid: "^0.5.0"
+  uri: "^0.11.0"
```
Comment (evanweible-wf) [inline pubspec.yaml]: This can be removed with the above comment

File: pubspec.yaml [hunk 3]
```diff
@@ -15,16 +15,16 @@ dependencies:
   browser: ">=0.10.0 <0.11.0"
   browser_detect: "^1.0.3"
   completion: "^0.1.5"
-  coverage: "^0.7.3"
+  coverage: "^0.7.7"
```
Comment (johnbland-wf) [inline pubspec.yaml]: Why isn't this bumping to the latest?

File: pubspec.yaml [hunk 4]
```diff
@@ -15,16 +15,16 @@ dependencies:
...
   dart_style: ">=0.2.4 <0.3.0"
```
Comment (johnbland-wf) [inline pubspec.yaml]: This is outdated as well.

**Comment by jayudey-wf** [comment]
closing as I don't know if it's still an issue

**Pull request closed without merging**